src/core/CMakeLists.txt

```diff
@@ -45,8 +45,7 @@ endif()
 
 # Allow CLOG to preprocess all the source files.
 CLOG_GENERATE_TARGET(core.clog ${SOURCES})
-
-target_link_libraries(core.clog PRIVATE inc)
+CLOG_ADD_ARGS(core.clog)
```

Comment:
Do we ever call `CLOG_GENERATE_TARGET` but not `CLOG_ADD_ARGS`? If not, we should have one wrapper function that calls clogs and does the `target_link_libraries`.

Reply:
We actually can't get lttng working on the Pi. That was actually my use case for disabling this. Same with mac, which doesn't have logging currently.

As for the function, I want to change that up a bit anyway.